utils/irqbalance/files/irqbalance.init

```diff
@@ -16,28 +16,44 @@ start_service() {
 	config_get_bool enabled irqbalance enabled 0
 	[ "$enabled" -gt 0 ] || return 0
 
-	# 2 is the default
 	config_get deepestcache irqbalance deepestcache 2
-
-	# 10 is the default
 	config_get interval irqbalance interval 10
-
-	# empty is the default
 	config_get banned_cpulist irqbalance banned_cpulist ''
+	config_get debug irqbalance debug 0
+
+	procd_open_instance "irqbalance"
+	if [ -n "$banned_cpulist" ]; then
+		procd_set_param env IRQBALANCE_BANNED_CPULIST="$banned_cpulist"
+	fi
+
+	procd_add_jail_mount_rw /proc
+	procd_set_param command /usr/sbin/irqbalance
+	procd_append_param command -f
 
-	# A list of IRQ's to ignore
-	banirq=""
+	# 2 is the default for deepestcache
+	if [ "$deepestcache" -ne 2 ]; then
+		procd_append_param command "-c $deepestcache"
+	fi
+
+	# 10 is the default for interval
+	if [ "$interval" -ne 2 ]; then
```

Comment:
I guess `2` here is a typo error. Is it a default value defined by us, or upstream irqbalance? Anyway, this if statement seems unnecessary.